src/bin/rustfmt.rs

```diff
@@ -63,8 +61,8 @@ enum Operation {
 struct CliOptions {
     skip_children: bool,
     verbose: bool,
-    write_mode: Option<WriteMode>,
-    file_lines: FileLines, // Default is all lines in all files.
+    write_mode: Option<String>,
+    file_lines: Option<String>,
```

Comment:
OK, I've gone with option 1. I had a play with some other ideas that were more like `Ref`, but they all introduced extra noise when accessing configuration.